user/enterprise/platform-tips.md

```diff
@@ -309,6 +309,28 @@ Additionally, please the following command to clean up all Docker build images:
 $ sudo docker images | grep travis | awk '{print $3}' | xargs sudo docker rmi -f
 ```
 
+## Find out maximum available concurrency
+
+If you wish to find out how much concurrency is available in your Travis CI Enterprise setup, please connect to your platform machine via ssh and follow these steps:
+
+```
+$ travis bash
+root@te-main:/# rabbitmqctl list_consumers -p travis | grep builds.trusty | wc -l
+```
+
+The number that's returned here is equal to the maximum number of concurrent jobs that are available. To adjust concurrency, please follow the instructions [here](/user/enterprise/worker-configuration/#configuring-the-number-of-concurrent-jobs) for each worker machine.
+
+## Find out how many worker machines are connected
+
+If you wish to find out how many worker machines are currently connected, please connect to your platform machine via ssh and follow these steps:
+
+```
+$ travis bash
+root@te-main:/# rabbitmqctl list_consumers -p travis | grep amq.gen- | wc -l
+```
+
+The number that's returned here is equal to the currently connected worker machines. If you need to spin up more worker machines, please follow the instructions [here](/user/enterprise/installation/#install-travis-ci-worker).
```

Comment:
Here I feel we could just delete 

> The number that's returned here is equal to the currently connected worker machines.

Reply:
If you can avoid it, instead of using `[here]` as the link text, use some more descriptive

> If you need to [spin up more worker machines](/user/enterprise/installation/#install-travis-ci-worker), see the docs on the [Installing workers](/user/enterprise/installation/#install-travis-ci-worker).